projects/plugins/jetpack/modules/custom-css/csstidy/class.csstidy.php

```diff
@@ -1,4 +1,5 @@
 <?php
+// phpcs:ignoreFile
```

Comment:
Is there a specific reason while you set to ignore the whole file? It's full of errors at the moment, but maybe we can stick to our coding standards for new code we introduce?

Reply:
Yeh, I wasn't sure how to handle it given it was original a 3rd party file, but given it is never going to be updated externally now you are probably right, so have removed the ignore.